src/client.js

```diff
@@ -4,6 +4,7 @@ var inherits = require('inherits'),
 	Promise = require('lie');
 
 var Port = require('./port'),
+	resizer = require('./plugins/iframe-resizer/client'),
```

Comment:
I would rather this not be brought in by the client by default, but that's a change I'll make on the client/host at the same time, later (that's step 3)